styles/styles.css

```diff
@@ -749,6 +749,15 @@ main .section.narrower > .tab-item.active > .section-container {
   max-width: 45rem;
 }
 
+/* section - large-padded */
+main .section.large-padded {
+  padding: 4rem 1rem;
+}
+
+main .section.large-padded h2 {
```

Comment:
How about we just keep the default `h2` style if the difference is acceptable? /cc @ehrro

Reply:
I thought the same, maybe we should merge the base typography of sunstar.com

Reply:
I am not sure if this site requires a typography update or it look good in its current state. If an update is needed, we can cherry-pick [this commit](https://github.com/hlxsites/sunstar/commit/0d8af249631ab6678337a8c4b8e47f13b3d45e53)
@ehrro wdyt?